less/home-page.less

```diff
@@ -56,20 +68,106 @@
       p {
         margin-bottom: 26px;
       }
-
-      .btn {
-      }
+      
+      
     }
 
     .walk-img {
       width: 46%;
       margin-left: 4%;
+      max-width: 100%;
       
       img {
-        width: 100%;
+       width: 100%;
         height: auto;
         border-radius: 10px;
       }
     }
+    
+  }
+   // added mixins to my button
+  .btn{
+    .button();
+    animation-name: spin;
+    animation-duration: 8s;
+
+  }
+  // added animation
+  @keyframes spin{
+    from{transform: rotate(0deg);}
+    to{transform: rotate(360deg);}
+}
+  
+}// home
+
+
+// media queries start here
+@media @mobile{
+  
+  .walk-img {
+     
+   width: 100%;
+    
+    
+   #walk-img{
+     width: 100%;
+   }
+  }
+    
+  .space-images #launch-img{
+    max-width: 500px;
+    /*max-height: 500px; */
+    width: 100%;
+    /*height: 100%; */
+    margin: 10px;
+    padding: 0;
+    
+    
+
+  }
+  
+  .space-images :nth-last-child(1){
+    display: none;
+  }
+  .space-images :nth-child(2){
+    display: none;
+  }
+  .walk-text{
+    width: 100%;
+    
+    min-width: 100%;
+    
+   
+
+    h3 {
+      margin-bottom: 10px;
+    }
+
+    p {
+      margin-bottom: 0px;
+    }
+     
+
+    
+  }
+  .cloned #clone{
+    width: 100%;
+    border-radius: 5%;
+    border-top: .5px solid grey;
+    padding: 10px 10px 10px 10px;
```

Comment:
The TRBL model that CSS uses accepts multiple argument syntax with 1, 2, 3, or 4(as you show here) arguments. The other 3 are 

1 Argument
``` CSS
padding: <ALL 4 Selectors>

|or|

padding: 10px
```

2 Arguments
``` CSS
padding: <Top/Bottom> <Right/Left>
```

3 Arguments
``` CSS
padding: <Top> <Right/Left> <Bottom>
```